svc/FetchService.js

```diff
@@ -66,13 +85,15 @@ export class FetchService {
             url = XH.baseUrl + url;
         }
 
+        headers = this.getHeaders(headers, contentType, url);
+
         // 2) Prepare merged options
         const defaults = {
                 method,
                 cors: true,
                 credentials: 'include',
                 redirect: 'follow',
-                headers: new Headers({'Content-Type': contentType})
+                headers: new Headers(headers)
```

Comment:
Won't these get overwritten all the time by the unmodified version off of opts? 🤔